### PR TITLE
Fogl 7495:  Fledge: Audit Log "NTFSN" generated even when delivery

### PR DESCRIPTION
**Pull request by Innovative-ashwin**

### CODE REVIEW

File: asset.cpp [hunk 1]
```diff
@@ -42,13 +42,12 @@ Asset::~Asset()
...
-void Asset::notify(const string& notificationName, const string& triggerReason, const string& message)
+bool Asset::notify(const string& notificationName, const string& triggerReason, const string& message)
 {
-vector<Datapoint *>	datapoints;
-
+	vector<Datapoint *>	datapoints;
 	if (!m_ingest)
 	{
-		return;
+		return false;
```
Comment (gnandan) [inline asset.cpp:50]: Can we log some message here why retuning false ?

File: asset.cpp [hunk 2]
```diff
@@ -42,13 +42,12 @@ Asset::~Asset()
  * @param triggerReason		Why the notification is being sent
  * @param message		The message to send
```
Comment (gnandan) [inline asset.cpp:43]: Add Return value

File: asset.cpp (hunk 1 above)
Comment (Innovative-ashwin) [reply asset.cpp:50]: fixed

**Review by MarkRiddoch — APPROVED** [review]
Please fix DCO

**Pull request merged**